mix.exs

```diff
@@ -38,16 +38,15 @@ defmodule Membrane.SRT.Mixfile do
   defp deps do
     [
       {:membrane_core, "~> 1.2.3"},
-      {:ex_libsrt,
-       github: "membraneframework-labs/ex_libsrt", ref: "6b331cc5e07c1a066c7bf53b14dedb91efe29e5d"},
+      {:ex_libsrt, github: "membraneframework-labs/ex_libsrt"},
       {:membrane_mpeg_ts_plugin, github: "kim-company/membrane_mpeg_ts_plugin"},
-      {:membrane_timestamp_queue, "~> 0.2.2"},
-      {:crc, "~> 0.10"},
-      {:membrane_aac_plugin, "~> 0.19.0"},
-      {:membrane_h26x_plugin, "~> 0.10.4"},
+      {:membrane_aac_plugin, "~> 0.19.0", optional: true},
+      {:membrane_h26x_plugin, "~> 0.10.4", optional: true},
```

Comment:
Do we need them now? I don't think we explicitly call functions from these deps.

Reply:
We use them in example (`sender.exs`). So it's more a deps policy question. I can add it to `Mix.install([...` for this example and delete it from here instead

Reply:
Indeed, it think it would be better to do it this way, that's how we do it in vast majority of places.